app/views/application/_navbar.html.erb

```diff
@@ -44,21 +44,19 @@
 <% if current_user && current_organization %>
 <nav role="navigation">
   <div class="container-fluid">
```

Comment:
@mllocs I think this should be "container", with "container-fluid" looks weird on desktop:

![captura de pantalla 2018-06-05 a las 18 44 25](https://user-images.githubusercontent.com/576701/40990336-c669a370-68f0-11e8-8045-6f5355dc94c5.png)